tests/specs/build-number.test.js

```diff
@@ -0,0 +1,19 @@
+import child_process from 'node:child_process';
+import fs from 'node:fs';
+import path from 'node:path';
+import process from 'node:process';
+
+import { test, expect } from 'vitest';
+
+test('cli behaviour for specific build', async function () {
+    // NW.js binary is installed via nwjs/npm-installer in CI before running this test
```

Comment:
All the other JS files have 2 space indentation. This file has 4. If you install the EditorConfig plugin, then your editor will automatically use the matching indentation for the repo as defined in `./.editorconfig`.

* https://editorconfig.org/#download